README.md

```diff
@@ -206,11 +211,11 @@ ui5: {
 ```
 
 Specific config options
-- [testpage](#testpage)
+- [test page](#testpage)
```

Comment:
Wouldn't it be better to use the config option name here, which is `testpage`?

Reply:
Yes.